CHANGES.rst

```diff
@@ -28,6 +34,8 @@ API Changes
 Cubeviz
 ^^^^^^^
 
+- ``get_data`` no longer supports ``function`` or ``spatial_subset`` as argue
```

Comment:
Seems incomplete.

Reply:
woah, that was weird.  Git must have staged before the file buffer was completely written or something 🤷.  Fixed now.